build.gradle

```diff
@@ -56,6 +59,21 @@ nexusIQScan {
   allConfigurations = true 
 }
 
+test {
+	useJUnitPlatform()
+    finalizedBy jacocoTestReport // report is always generated after tests run
```

Comment:
I think eventually we should not be including the report in git. We should add it to .gitignore and the pipeline should run the tests and generate a CLI-friendly report. Thoughts? Beyond the scope of this PR, but wanted to see if we were on the same page.

Reply:
At the moment the report gets created under the build directory which is excluded by .gitingore.

I completely agree on the pipeline improvements, we should have something good in the CLI, let's have a different PR for that.

Reply:
My mistake!